### PR TITLE
switch neuvector api and web to tls, change domain to platform.hmcts.net

### PR DESCRIPTION
**Pull request by danielwilsonkainos**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-12488

### Change description ###

switch neuvector api and web to tls, change domain to platform.hmcts.net

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/neuvector/neuvector/aat/00.yaml [hunk 1]
```diff
@@ -75,4 +80,8 @@ spec:
               Cluster_Name: aat00
       manager:
         ingress:
-          host: neuvector00.service.core-compute-aat.internal
+          host: neuvector00.aat.platform.hmcts.net
```
Comment (reespozzi) [inline apps/neuvector/neuvector/aat/00.yaml:83]: Should this be `cft-neuvector00.aat.platform.hmcts.net` as in the 01 file?

File: apps/neuvector/neuvector/aat/00.yaml [hunk 2]
```diff
@@ -32,7 +32,12 @@ spec:
           secretName: nv-storage-secret
           shareName: neuvector-data-00
         ingress:
-          host: neuvector00.service.core-compute-aat.internal
+          enabled: true
```
Comment (adusumillipraveen) [inline apps/neuvector/neuvector/aat/00.yaml:35]: We don't need ingress for api here, its needed only on 01 for a specific use case.

File: apps/neuvector/neuvector/aat/00.yaml (hunk 1 above)
Comment (adusumillipraveen) [inline apps/neuvector/neuvector/aat/00.yaml:83]:
```suggestion
          host: cft-neuvector00.aat.platform.hmcts.net
```

File: apps/neuvector/neuvector/aat/00.yaml (hunk 2 above)
Comment (danielwilsonkainos) [reply apps/neuvector/neuvector/aat/00.yaml]: removed

File: apps/neuvector/neuvector/aat/00.yaml [hunk 3]
```diff
@@ -75,4 +73,8 @@ spec:
               Cluster_Name: aat00
       manager:
         ingress:
-          host: neuvector00.service.core-compute-aat.internal
+          host: cft-neuvector00.aat.platform.hmcts.net
+          enabled: true
```
Comment (thomast1906) [inline apps/neuvector/neuvector/aat/00.yaml:77]: aat/00.yaml & aat/01.yaml are not similar? 

01.yaml for example has `path: "/"` etc

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**